workflows/flows/copy_amplicon_pipeline_results.py

```diff
@@ -0,0 +1,15 @@
+from prefect import flow
+
+from workflows.prefect_utils.slurm_flow import move_data
+
+
+@flow(
+    name="Copy Amplicon Pipeline Results",
+    flow_run_name="Copy Amplicon Pipeline Results",
+)
+async def copy_amplicon_pipeline_results():
```

Comment:
according to RP #34 should it still be `async`?

Reply:
think I'll keep this as a normal function, unless the need for making it async arises